### PR TITLE
feat: added price filter

### PR DESCRIPTION
**Pull request by AKutusenko**
## Describe your changes

- Added to property filter inputs for min and max price 
- Added remove filters button

image

## Issue ticket code (and/or) and link

- [Link to JIRA ticket](https://testhomework4.atlassian.net/jira/software/projects/AG/boards/6?assignee=5fd26b3708757f006e4d61d4&selectedIssue=AG-231)

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [x] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [ ] No inline styles
- [x] Imports are absolute
- [x] Attach a screenshot if PR has visual changes.

### CODE REVIEW

**Review by rudenkoserhii — APPROVED** [review]

File: src/utils/property-filters/getMainPagePropertyFilter.ts [hunk 1]
```diff
@@ -22,6 +34,13 @@ const getMainPagePropertyFilter = (formData: PropertyFilterFormData): string =>
         district: location?.searchParams.district ?? '',
       })
     );
+  if (rent === 'for_sale') {
```
Comment (canterbery) [inline src/utils/property-filters/getMainPagePropertyFilter.ts]: We have a enum for that.  
Check QobrixPropertySaleRent enum

File: src/utils/property-filters/getMainPagePropertyFilter.ts [hunk 2]
```diff
@@ -22,6 +34,13 @@ const getMainPagePropertyFilter = (formData: PropertyFilterFormData): string =>
...
+    filter = filter.concat(getSellingPriceFilter(priceRangeSaleFrom, priceRangeSaleTo));
+  }
+  if (rent === 'for_rent') {
```
Comment (canterbery) [inline src/utils/property-filters/getMainPagePropertyFilter.ts]: Same

File: src/utils/property-filters/getMainPagePropertyFilter.ts (hunk 1 above)
Comment (AKutusenko) [reply src/utils/property-filters/getMainPagePropertyFilter.ts]: fixed

File: src/utils/property-filters/getMainPagePropertyFilter.ts (hunk 2 above)
Comment (AKutusenko) [reply src/utils/property-filters/getMainPagePropertyFilter.ts]: fixed

**Review by canterbery — APPROVED** [review]

File: src/locales/langs/en.json [hunk 3]
```diff
@@ -358,6 +359,11 @@
       "bedrooms": "Bedrooms",
       "apply": "Apply filters",
       "locationsPlaceholder": "Location search",
+      "priceRangeRentFromPlaceholder": "Min",
+      "priceRangeRentToPlaceholder": "Max",
+      "priceRangeSaleFromPlaceholder": "Min",
+      "priceRangeSaleToPlaceholder": "Max",
+      "removeFilters": "Remove filters",
```
Comment (lyudad) [inline src/locales/langs/en.json]: `Clear filter` change text please

Comment (AKutusenko) [reply src/locales/langs/en.json]: fixed

**Pull request merged**